python/pyspark/pandas/namespace.py

```diff
@@ -2797,7 +2803,7 @@ def to_numeric(arg):
     1    1.0
     2    2.0
     3   -3.0
-    dtype: float32
+    dtype: float64
```

Comment:
do you know why the type of result changes in this case? from a cursory look, it should be the same.

Reply:
You're right. It should be `float32`.

I changed it from previous fix, and didn't reverted mistakenly.